client/src/components/Planning/PlanningData.ts

```diff
@@ -0,0 +1,103 @@
+//Mock data for Planning Graphs
```

Comment:
This file doesn't need to be a TypeScript file as the frontend project is not a TypeScript project